docs/tables/aws_ec2_instance.md

```diff
@@ -31,7 +30,6 @@ where
   monitoring_state = 'disabled';
 ```
 
-
```

Comment:
Did we need to remove these lines? What do other docs examples have?

Reply:
we always provide line space after query and comment , for other doc example also have same .